### PR TITLE
Feature/issue/222/todaybutton

### PR DESCRIPTION
**Pull request by soph-em**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. Additionally, please include reasoning if tests have failed.

Adds today button from issue https://github.com/Together-100Devs/Together/issues/222 

Please select everything applicable. Please, do not delete any lines.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change requires an update to testing

## Issue

- [ ] Is this related to a specific issue? If so, please specify: #222 

# Checklist:

- [x] This PR is up to date with the main branch, and merge conflicts have been resolved
- [ ] I have executed `npm run test` and all tests have passed successfully or I have included details within my PR on the failure.
- [ ] I have executed `npm run lint` and resolved any outstanding errors. Most issues can be solved by executing `npm run format`
- [X] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by Natashka333** [comment]
Great job @soph-em ! Thank you for contribution! The only one thing which I noticed is the Today button pushes September month into two lines. Potential solutions can be: giving up on logo and locate Today button left from the month; or keep it on the right but push it more to the right. I think, I am more for first option. What do you think @soph-em and @Caleb-Cohen ?

File: client/src/features/calendar/MonthAndYear.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ const MonthAndYear = ({
           </svg>
         </button>
       </div>
-      <h2 className="ml-2 w-[16ch] text-3xl font-bold leading-none">
+      <h2 className="ml-2 w-[12ch] text-3xl font-bold leading-none">
```
Comment (Caleb-Cohen) [inline client/src/features/calendar/MonthAndYear.js:45]: This change seems to force September to create a new line. Changing this to 14 seems to correct the issue.

File: .eslintrc.js [hunk 2]
```diff
@@ -33,10 +33,10 @@ module.exports = {
...
       },
     ],
-    "linebreak-style": [
-      "error",
-      process.platform === "win32" ? "windows" : "unix",
-    ],
+    // "linebreak-style": [
+    //   "error",
+    //   process.platform === "win32" ? "windows" : "unix",
+    // ],
```
Comment (Caleb-Cohen) [inline .eslintrc.js:39]: Could you revert this? It's not ideal at the moment, but we'll need to find a proper fix to it.

using `--no-verify` to a commit should bypass the linting requirement on windows.

**Review by Caleb-Cohen — CHANGES_REQUESTED** [review]
Added some comments.

File: client/src/features/calendar/MonthAndYear.js (hunk 1 above)
Comment (Caleb-Cohen) [reply client/src/features/calendar/MonthAndYear.js:45]: @Natashka333 this should satisfy your feedback.

Comment (soph-em) [reply client/src/features/calendar/MonthAndYear.js]: thank you for finding that - have resolved that in commit [6e88062](https://github.com/Together-100Devs/Together/pull/388/commits/6e88062d453ef1b54be70f6191096d15481a7013)

File: .eslintrc.js (hunk 2 above)
Comment (soph-em) [reply .eslintrc.js]: sorry thought I undid that with my merges! have undone it in [6e88062](https://github.com/Together-100Devs/Together/pull/388/commits/6e88062d453ef1b54be70f6191096d15481a7013)

**Pull request merged**